pkg/cli/cmds/root.go

```diff
@@ -83,8 +83,8 @@ var (
 			Destination: &config.CloudProviderConfig,
 		},
 		&cli.StringFlag{
-			Name:   "profile",
-			Usage:  "(security) Validate system configuration against the selected benchmark (valid items: " + rke2.CISProfile123 + " )",
+			Name: "profile",
+			Usage: "(security) Validate system configuration against the selected benchmark (valid items: cis, cis-1.23 (deprecated))"),
```

Comment:
did we loose the formatting here? I think `go fmt` would complain about this indentation.

Reply:
oh I see.
```suggestion
			Name: "profile",
			Usage: "(security) Validate system configuration against the selected benchmark (valid items: cis, cis-1.23 (deprecated))",
```